sparkmagic/sparkmagic/utils/constants.py

```diff
@@ -1,9 +1,11 @@
 # Copyright (c) 2015  aggftw@gmail.com
 # Distributed under the terms of the Modified BSD License.
 
+import os
 
-HOME_PATH = "~/.sparkmagic"
-CONFIG_FILE = "config.json"
+
+HOME_PATH = os.environ.get("SPARKMAGIC_CONF_DIR", "~/.sparkmagic")
```

Comment:
Looks good. You just need to rev the minor version of the 3 packages in the repo:

https://github.com/jupyter-incubator/sparkmagic/blob/master/autovizwidget/autovizwidget/__init__.py
https://github.com/jupyter-incubator/sparkmagic/blob/master/hdijupyterutils/hdijupyterutils/__init__.py
https://github.com/jupyter-incubator/sparkmagic/blob/master/sparkmagic/sparkmagic/__init__.py